templates/partials/summary/list-summary.html

```diff
@@ -3,7 +3,7 @@
 {% if list.editable %}
   {% set rows = map_list_collector_config(
     list.list_items,
-    "person",
+    True,
     _("Change"),
     _("Change details for {item_name}"),
     _("Remove"),
```

Comment:
Minor and existing, I suggest keyword args.

Reply:
Added kwargs in this tempate 👍